lib/testLauncher/ipc/ipcHelper.js

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
You implemented own client/server for IPC. I'm curios if it would be possible to use NodeJS built-in IPC, as described here:
https://nodejs.org/dist/latest-v11.x/docs/api/child_process.html
Is there a benefit from using raw net?

Reply:
Decided to implement our own, because we spawn processes.